curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153a3ebb4f7f05b8401b716.md

```diff
@@ -7,45 +7,22 @@ dashedName: step-21
 
 # --description--
 
-Resize the preview to see the layout change from your media query.
+To complete this challenge, add `alt` text to the images.
 
-Finally, create another media query for screens smaller than `600px` wide. In that media query, create a `#gallery img` rule and set the `width` property to `100%`. This will give your gallery a single-column layout.
+The alternative text should describe the image content. Screen readers announce alternative text in place of images. If the image can't be loaded, or is blocked, the alternative text is presented in place of the image.
 
-Your CSS Flexbox Photo Gallery is now complete.
+Add an `alt` attribute to all nine images and give them text describing each. Use at least `5` letters for each `alt` text value.
 
-# --hints--
-
-You should have a second `@media` query.
-
-```js
-assert(new __helpers.CSSHelp(document).getCSSRules('media')?.length === 2);
-```
-
-Your new `@media` query should come after your existing one. You should have a `@media (max-width: 800px)` rule.
-
-```js
-assert(new __helpers.CSSHelp(document).getCSSRules('media')?.[0]?.media?.mediaText === '(max-width: 800px)');
-```
-
-Your new `@media` query should have a `max-width` of `600px` like this: `@media (max-width: 600px)`.
-
-```js
-assert(new __helpers.CSSHelp(document).getCSSRules('media')?.[1]?.media?.mediaText === '(max-width: 600px)');
-```
-
-Your new `@media` query should have a `#gallery img` selector.
+**Note:** If you are unable to view the images, you may use any `alt` text you like.
```

Comment:
```suggestion
```
I don't think we need to explicitly say this.

Reply:
That is what was recommended by @bbsmooth

It is in case the student is visually impaired and can not see the images. Refer to #46639

Reply:
I saw the suggestion. I don't think we need to explicitly add this for the visually impaired. Anyone can use any text, not just visually impaired. So if we just say to add any text - they should get it. I added that they are "cat photos" in the suggestion on line before this so they know what they are.